src/NzbDrone.Core/Parser/Model/MediaInfoModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using NzbDrone.Core.Datastore;
+
+namespace NzbDrone.Core.Parser.Model
+{
+    public class MediaInfoModel : IEmbeddedDocument
+    {
+        public string AudioFormat { get; set; }
+        public int AudioBitrate { get; set; }
+        public int AudioChannels { get; set; }
+        public int SchemaRevision { get; set; }
```

Comment:
Can we add Bits and SampleRate here. Taglib provides both I believe. Bits should be used to identify certain qualities (24bit Flac).